circle.yml

```diff
@@ -0,0 +1,17 @@
+machine:
+  node:
+    version: v0.12.16
```

Comment:
Do we want have fixed version here? I would keep it open for patch versions. On specific varsion it will be tested in core app.